packages/mobile/src/send/SendAmount.tsx

```diff
@@ -206,6 +215,21 @@ export class SendAmount extends React.Component<Props, State> {
     }
   }
 
+  isPaymentLimitReached = (now: number) => {
+    // we are only interested in the last 24 hours
+    const paymentsLast24Hours = this.props.recentPayments.filter(
+      (p: PaymentInfo) => timeDeltaInHours(now, p.timestamp) < 24
+    )
+
+    const initial = new BigNumber(this.state.amount)
+    const amount: BigNumber = paymentsLast24Hours.reduce(
+      (sum, p: PaymentInfo) => sum.plus(p.amount),
+      initial
+    )
+
+    return amount <= DAILY_PAYMENT_LIMIT_USD
```

Comment:
i'ts reached if amount is greater than DAILY_LIMIT

 Doesnt this return true if amount is less than or equal to DAILY_LIMIT ?

Reply:
Yeah, I think this is the inverse of the function name

Reply:
good catch! just testing you guys :) jk